

include/asm/smp.h:55:1: warning: "cpu_possible" redefined
include/asm/smp.h:54:1: warning: "cpu_online" redefined

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-power4-akpm/include/asm-ppc64/smp.h |    2 --
 1 files changed, 2 deletions(-)

diff -puN include/asm-ppc64/smp.h~ppc64-cpu_online-fix include/asm-ppc64/smp.h
--- 25-power4/include/asm-ppc64/smp.h~ppc64-cpu_online-fix	2004-06-09 00:02:16.557661648 -0700
+++ 25-power4-akpm/include/asm-ppc64/smp.h	2004-06-09 00:02:45.172311560 -0700
@@ -51,8 +51,6 @@ extern cpumask_t cpu_possible_map;
 extern cpumask_t cpu_available_map;
 
 #define cpu_present_at_boot(cpu) cpu_isset(cpu, cpu_present_at_boot)
-#define cpu_online(cpu)          cpu_isset(cpu, cpu_online_map) 
-#define cpu_possible(cpu)        cpu_isset(cpu, cpu_possible_map) 
 #define cpu_available(cpu)       cpu_isset(cpu, cpu_available_map) 
 
 /* Since OpenPIC has only 4 IPIs, we use slightly different message numbers.
_
